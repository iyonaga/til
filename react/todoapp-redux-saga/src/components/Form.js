import React, { Component } from 'react';
import './Form.css';

export default class Form extends Component {
  render() {
    return (
      <div>
        <input type="text" className="input" onKeyDown={(e) => this.props.onInputKeyDown(e)} />
      </div>
    )
  }
}
