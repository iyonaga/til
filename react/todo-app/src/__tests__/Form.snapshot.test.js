import React from 'react';
import Form from '../Form';
import renderer from 'react-test-renderer';

describe('Form', () => {
  it('Form renders correctly', () => {
    const tree = renderer.create(<Form/>).toJSON();
    expect(tree).toMatchSnapshot();
  })
})
