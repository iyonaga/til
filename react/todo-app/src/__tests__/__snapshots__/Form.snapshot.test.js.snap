// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Form Form renders correctly 1`] = `
<div>
  <input
    className="input"
    onKeyDown={[Function]}
    type="text"
  />
</div>
`;
